tools/common_lib/src/gemm.h

```diff
@@ -1318,6 +1342,7 @@ class GemmUmdD3d12Dispatcher : public GemmBaseDispatcher
 
     dnnl::memory::desc input_a_memory_desc_;
     dnnl::memory::desc input_b_memory_desc_;
+    dnnl::memory::desc input_b_memory_desc;
```

Comment:
is it needed?

Reply:
Previously, the variable was getting declared initialized outside of an if-else block so we didn't need to declare it in the class variables.
However, now it being declared inside an if-else block so we need to declare it.
I had thought about using input_b_memory_desc_ instead, but it is being used to get another memory descriptor, so rather than overwriting the previous value - I decided to declare a new variable in the class variables.

Reply:
Please use only `input_b_memory_desc_`, it's fine to use it before getting that "another" descriptor.
Also I think you are missing some code in function void initialize(...) which defines input format for B tensor in managed cases

Reply:
Got it - will use the input_b_memory_desc_ variable itself.